src/main/java/com/github/leroyguillaume/keycloak/bcrypt/BCryptPasswordHashProvider.java

```diff
@@ -68,13 +53,19 @@ public void close() {
     }
 
     @Override
-    public boolean verify(String rawPassword, CredentialModel credential) {
-        return BCrypt.checkpw(rawPassword, credential.getValue());
+    public boolean verify(String rawPassword, PasswordCredentialModel credential) {
+        return BCrypt.checkpw(rawPassword, credential.getPasswordSecretData().getValue());
```

Comment:
One day, I'll replace this library by https://github.com/patrickfav/bcrypt which is more complete...